io.github.quodlibet.ExFalso.yaml

```diff
@@ -0,0 +1,100 @@
+---
+app-id: io.github.quodlibet.ExFalso
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+command: exfalso
+build-options:
+  no-debuginfo: true
+  strip: true
+finish-args:
+  - "--share=network"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--filesystem=host"
+
+  # some embedded covers and the fifo is still shared through /tmp
+  - "--filesystem=/tmp"
+
+  # bookmarks in the filesystem browser
+  - "--filesystem=xdg-config/gtk-3.0:ro"
+
+  # ...detecting the running desktop environment
+  - "--talk-name=com.canonical.Unity.Launcher"
+  - "--talk-name=org.enlightenment.wm.service"
+  - "--talk-name=org.kde.plasmashell"
+
+  # ...various other plugins
+  - "--talk-name=org.freedesktop.FileManager1"
+  - "--talk-name=org.gnome.SessionManager"
+  - "--talk-name=org.xfce.FileManager"
+
+  # ...talking to dconf
+  - "--filesystem=xdg-run/dconf"
+  - "--filesystem=~/.config/dconf:ro"
+  - "--talk-name=ca.desrt.dconf"
+  - "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+
+modules:
+  - python-modules.yaml
+  - flathub-shared-modules/dbus-glib/dbus-glib-0.110.json
+  - name: kakasi
+    sources:
+      - type: archive
+        url: http://kakasi.namazu.org/stable/kakasi-2.3.6.tar.xz
+        sha256: 2ee57b1b03c74fd5766e741c3812028efc73bc0e0bf93a6e7ff20eb4701f3ee3
+    cleanup:
+      - /lib
+      - /include
+  - name: chromaprint
+    buildsystem: cmake
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+      - "-DBUILD_TESTS=off"
+      - "-DBUILD_TOOLS=off"
+      - "-DAUDIO_PROCESSOR_LIB=dummy"
+    sources:
+      - type: archive
+        url: https://github.com/acoustid/chromaprint/releases/download/v1.4.3/chromaprint-1.4.3.tar.gz
+        sha256: ea18608b76fb88e0203b7d3e1833fb125ce9bb61efe22c6e169a50c52c457f82
+  - name: gst-libav
```

Comment:
Already in the runtime

Reply:
The runtime version doesn't include WMA support